test/extended/cluster/config/master-vert.yaml

```diff
@@ -0,0 +1,60 @@
+provider: local
```

Comment:
I can move the config to the testdata, but I can't use FixturePath since the config is passed as a flag to viper?